packages/core/src/Machine.ts

```diff
@@ -26,26 +25,16 @@ export function Machine<
   config: MachineConfig<TContext, TStateSchema, TEvent>,
   options?: Partial<MachineOptions<TContext, TEvent>>,
   initialContext?: TContext
```

Comment:
overloads here still mention `initialContext` parameter, but it's no longer supported